src/node.cc

```diff
@@ -4202,8 +4208,14 @@ int Start(int argc, char** argv) {
   V8::SetEntropySource(crypto::EntropySource);
 #endif
 
-  const int thread_pool_size = 4;
-  default_platform = v8::platform::CreateDefaultPlatform(thread_pool_size);
+  int pool_size = v8_default_thread_pool_size;
+
+  if (v8_thread_pool_size != nullptr) {
+    // if the parameter is given, try to convert the string to an int.
+    pool_size = atoi(v8_thread_pool_size);
+  }
+
+  default_platform = v8::platform::CreateDefaultPlatform(pool_size);
```

Comment:
I would simply say that if the value is not within an acceptable range, the nearest acceptable value is selected... essentially, anything less than 0 is treated as 0, anything greater than the current max is treated as the max. If we add that to the documentation, no warning should be necessary.